---
title: BAPG 2022
---

## UC Berkeley, 10/15/22

We are very excited to announce the 20th Bay Area Population Genomics (BAPG) Conference at UC Berkeley! Thank you to everyone who has signed up to present either a talk or a poster. We have posted the speakers, talk titles, and poster presenters on the schedule below.

Thank you to our sponsors Ancestry, Variant Bio, D2G Oncology, and Personalis for helping us put on this great conference!

### Information

**Location**: Stanley Hall, UC Berkeley

**Time**: 9:30am - 5pm, Saturday, October 15 (Breakfast and check-in begins at 8:30am)

**Transportation to UC Berkeley**: For those planning on driving to Berkeley, visitor parking is available at a number of campus-owned parking lots. The closest ones to Stanley Hall will be the La Loma Parking Structure, the Lower Hearst Parking Structure, and the Foothill Lot. Visitor parking on the weekends is $2 per hour. See Berkeley's visitor parking page for more details. Please follow all posted signs and directions for campus parking lots, which all look something like this. For those looking to take public transportation, BART stops one block west of campus at the Downtown Berkeley station. From there, it is a very scenic walk up through campus to Stanley.

### Schedule

**8:30-9:30am**: Check-in and breakfast, atrium of Stanley Hall

**9:30-10:30**: Session 1
  * 9:30-9:45: Silu Wang, Coop Lab, UC Davis, "On the synthesis of Recombination Effect, Large X Effect, Snowball Effect underlying speciation"
  * 9:45-10:00: Frederik Filip Vinggaard Staeger, Section for Computational Biology, University of Copenhagen, "The unique genetic architecture of common diseases in Greenland"
  * 10:00-10:15: Elise Kerdoncuff, Moorjani Lab, UC Berkeley, "Reconstructing patterns of admixture through time: Insights from whole genome sequencing of South Asian populations"
  * 10:15-10:30: Nicolas Lou, Sudmant Lab, UC Berkeley, "Time-series data reveal the genomic ramifications of a fishery collapse"
  * 10:30-11:00 Coffee Break, Stanley Hall Atrium
  * 11:00-12:00 Session 2
  * 11:00-11:15: Tristram Dodge, Schumer Lab, Stanford, "The genetic architecture of sexual mimicry in swordtail (Xiphophorus) fishes"
  * 11:15-11:30: Lenore Pipes, Nielsen Lab, UC Berkeley, "Estimating the relative proportions of SARS-CoV-2 haplotypes from wastewater samples"
  * 11:30-11:45: Xiran Liu, Rosenberg Lab, Stanford, "A Dirichlet model of alignment cost in mixed-membership clustering results of ancestry inference"
  * 11:45-12:00: Serena Caplins, Bay Lab, UC Davis, and Williams Lab, UC Berkeley,  "A life-history polyphenism along temporally and geographically shifting environmental factors in the sea slug Alderia willowi"

**12:00-1:30**: Lunch, Stanley Hall Atrium

**1:30-2:30**: Session 3
  * 1:30-1:45: Joao Ascensao, Hallatschek Lab, UC Berkeley, "Adaptive landscapes after diversification: what do they look like and what do they do?"
  * 1:45-2:00: Zhiru Liu, Good Lab, Stanford, "Dynamics of bacterial recombination in the human gut microbiome"
  * 2:00-2:15: Hakhamanesh Mostafavi, Pritchard Lab, Stanford, "Limited overlap of eQTLs and GWAS hits due to systematic differences in discovery"
  * 2:15-2:30: William Dewitt, Song Lab, UC Berkeley, "Evolutionary dynamics of antibody affinity maturation in replica germinal centers"
 
**2:30-3:00**: Keynote Talk Chuck Langley, UC Davis, "Haplotypes spanning the centromere proximal regions of Drosophila and humans"

**3:00-5:00**: Poster Session and Happy Hour, Stanley Hall 1st Floor Atrium
  * Alexander Ioannidis - "Ancestry-specific methods in population genetics"
  * Allison Gaudinier - "Characterizing Mimulus guttatus adaptation to serpentine soil"
  * Anastasia Lyulina - "Linkage equilibrium between rare alleles"
  * Andrew Vaughn - "Bayesian inference of admixture graphs on Native American and Arctic populations"
  * Daniel Wong - "Quantifying the adaptive landscape of commensal gut bacteria using high-resolution lineage tracking"
  * Diana Aguilar - "Understanding strawberry poison frog color polymorphism in Bocas del Toro"
  * James Ferrare - "Evolution of Evolvability in Adapting Populations"
  * Kaleda Denton - "Re-examining the evolution of altruism"
  * Maike Morrison - "FSTruct: An Fst-based tool for quantifying ancestry variability"
  * Marianna Karageorgi - "Adaptive tracking in response to seasonal insecticide pressure in Drosophila"
  * Matthew Aguirre - "Simulating effects from genetic and experimental perturbations to gene regulatory networks"
  * Maya Lemmon-Kishi - "A Penalized Likelihood Approach for Estimating Haplotypes from Environmental DNA"
  * Nico Ayala - "Inferring Non-Additive Multi-Locus Selection in Admixed Populations Using Hidden Markov Model"
  * Noah Simon - "Evidence for polygenic, directional cis-regulatory evolution in Southeast Asian mice"
  * Olivia Ghosh - "Emergent evolutionary forces in spatial models of luminal growth in the human gut microbiota"
  * Ren Hamm - "Evolutionary dynamics of stress-activated mobile elements in Mimulus guttatus"
  * Sophie Jean Walton - "Within-host evolution and strain dynamics in the human gut microbiome during and after antibiotic perturbations"
  * Sylvia Durkin - "Gene regulatory basis of parallel environmental adaptation in house mice (Mus musculus domesticus)"
  * Yulin Zhang - "Recovering signatures of ghost admixture using ancestral recombination graphs"
  * Yun Deng - "A new framework for efficiently inferring ancestral recombination graph"
  * Yuval Simons - "Simple scaling laws control the architectures of complex traits"
